app/lib/frontend/handlers/experimental.dart

```diff
@@ -96,6 +96,7 @@ class ExperimentalFlags {
   bool get isDarkModeDefault => isEnabled('dark-as-default');
 
   bool get showDownloadCounts => isEnabled('download-counts');
+  bool get showDownloadCountsChart => isEnabled('download-counts-chart');
```

Comment:
How about calling it `download-counts-version-chart`?

Reply:
I have no strong preference, so I'll change it :-)